osidb/tests/factories.py

```diff
@@ -674,7 +674,9 @@ def vector(self):
         return vectors[self.version]
 
     comment = factory.LazyAttribute(
-        lambda o: "CVSS RH comment" if o.issuer == CVSS.CVSSIssuer.REDHAT else ""
+        lambda o: "CVSS RH comment"
+        if o.issuer == CVSS.CVSSIssuer.REDHAT and o.version == CVSS.CVSSVersion.VERSION3
+        else ""
```

Comment:
I will move this into a new PR because more things need to be changed: `comment` should be directly in the `FlawCVSS` model as BZ enables it only for CVSSv3 assigned by Red Hat. Affects cannot have comments in BZ.